### PR TITLE
Create RTSPtoWeb.yaml

### PR DESCRIPTION
**Pull request by thedroidkid**
Added RTSPtoWeb app

### CODE REVIEW

File: Apps/RTSPtoWeb/RTSPtoWeb.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        network_mode: bridge
+        ports:
+            - target: 8083
+              published: "8083"
+              protocol: tcp
+        restart: unless-stopped
+        volumes:
+            - type: bind
+              source: /DATA/AppData/rtsp-to-web
+              target: /tmp
```
Comment (dragonfire1119) [inline Apps/RTSPtoWeb/RTSPtoWeb.yaml:21]: You would need to add

```
x-casaos: # CasaOS specific configuration
      volumes:
        - container: /tmp
          description:
            en_us: "Container Path: /tmp"
        - container: /app/public/icons
          description:
            en_us: "Container Path: /app/public/icons"
      ports:
        - container: "8083"
          description:
            en_us: "Container Port: 8083"
```

File: Apps/RTSPtoWeb/RTSPtoWeb.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+x-casaos:
```
Comment (dragonfire1119) [inline Apps/RTSPtoWeb/RTSPtoWeb.yaml:22]: Can you look at: https://github.com/bigbeartechworld/big-bear-casaos/blob/8e00ebbee8eb936f1a44f888beea8a26570d06d1/Apps/homarr/docker-compose.yml#L49

**Review by dragonfire1119 — CHANGES_REQUESTED** [review]
Can you add this app to the README.md app list?

**Comment by thedroidkid** [comment]
The problem with this is config.json is stored in /dist/config.json but when I try setting the volume /DATA/AppData/rtsp-to-web to /dist the container will not start no matter what I do. For now I just mapped it to /tmp cuz I just manually copy the config over its a real pain

**Comment by dragonfire1119** [comment]
@thedroidkid Yeah, it needs to be made easier to install. I'll look at it more when I have time.

**Pull request closed without merging**